bluetooth/src/androidMain/kotlin/device/DefaultDeviceConnectionManager.kt

```diff
@@ -341,6 +325,8 @@ private val Int.gattStatusAsString get() = when (this) {
     GATT_CONNECTION_CONGESTED -> "ERROR_CONNECTION_CONGESTED"
     GATT_CONNECTION_TIMEOUT -> "ERROR_CONNECTION_TIMEOUT"
     GATT_FAILURE -> "ERROR_FAILURE"
+    19 -> "REMOTE_USER_TERMINATED_CONNECTION"
+    133 -> "DEVICE_NOT_FOUND"
```

Comment:
ah the famous 133.. I'm actually curious what's your source for "DEVICE_NOT_FOUND"?

Reply:
I found it in several posts while googling. nothing official though